src/test/java/com/zufar/icedlatte/order/converter/OrderDtoConverterTest.java

```diff
@@ -0,0 +1,104 @@
+package com.zufar.icedlatte.order.converter;
+
+import com.zufar.icedlatte.openapi.dto.OrderItemResponseDto;
+import com.zufar.icedlatte.openapi.dto.OrderResponseDto;
+import com.zufar.icedlatte.order.entity.Order;
+import com.zufar.icedlatte.order.entity.OrderItem;
+import com.zufar.icedlatte.order.stub.OrderDtoTestStub;
+import com.zufar.icedlatte.product.converter.ProductInfoDtoConverter;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static com.zufar.icedlatte.order.stub.OrderDtoTestStub.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+@SpringBootTest(classes = {OrderDtoConverterTest.Config.class})
+class OrderDtoConverterTest {
+
+    @Autowired
+    ProductInfoDtoConverter productInfoDtoConverter;
+    @Autowired
+    OrderItemDtoConverter orderItemDtoConverter;
+    @Autowired
+    OrderDtoConverter orderDtoConverter;
+
+    @Configuration
```

Comment:
Use @TestConfiuration if you want to create some additional beans needed for tests. By the way in this context you dont even need it since you use componentModel=Spring in mappers that makes them as spring beans automatically.